ctrd/utils.go

```diff
@@ -10,13 +11,33 @@ import (
 	"github.com/alibaba/pouch/apis/types"
 	"github.com/alibaba/pouch/pkg/errtypes"
 
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/oci"
 	"github.com/containerd/containerd/remotes"
 	"github.com/containerd/containerd/remotes/docker"
+	"github.com/containerd/containerd/runtime/linux/runctypes"
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 )
 
+// NewDefaultSpec new a template spec with default.
+func NewDefaultSpec(ctx context.Context, id string) (*specs.Spec, error) {
```

Comment:
We do not need default spec from containerd now

Reply:
will do.

Reply:
done.